src/cromshell/utilities/HTTPUtils.py

```diff
@@ -0,0 +1,21 @@
+import tempfile
+import requests
+from cromshell.utilities import IOUtils
+
+
+def assertCanCommunicateWithServer(cromwell_server):
+    with tempfile.TemporaryFile() as f:
```

Comment:
We've talked about this while we are going over this in person, but do we need such temporary files any more? The bash implementation needs this so that the temporary files could be parsed later, but python is much more powerful in this area that maybe the response can be just stored in `curl_out` define on line 9 and returned for a caller to be used/parse.

Reply:
done, removed temp files